types.go

```diff
@@ -293,3 +294,90 @@ func (m Metadata) GetObject(key string) (Metadata, error) {
 	out, _ := val.Value.(Metadata)
 	return out, nil
 }
+
+// AnnotationsMap defines a map of string arrays that contain the annotations data
+type AnnotationsMap map[string][]string
+
+// AnnotationKeyMaxSize defines the max size of an annotation key in bytes
+const AnnotationKeyMaxSize = 64
```

Comment:
you can probably remove these now if you want

Reply:
Unless i'm mistaken, wouldn't it be possible that a string smaller than 64 chars be bigger than 64 bytes? Thanks to unicode